components/browser/menu/src/main/java/mozilla/components/browser/menu/BrowserMenu.kt

```diff
@@ -90,12 +90,20 @@ class BrowserMenu internal constructor(
 
     enum class Orientation {
         UP,
-        DOWN
+        DOWN,
+        // Chooses the best orientation for a popup menu. It will appear below the anchor view if there is room,
+        // or above it if there is not.
+        DYNAMIC
```

Comment:
If you turn this comment into KDoc then it will show up here:
https://mozac.org/api/mozilla.components.browser.menu/-browser-menu/-orientation/

Reply:
👍